test/e2e/testnet/test_helpers.go

```diff
@@ -9,3 +10,13 @@ func NoError(message string, err error) {
 		log.Fatalf("%s: %v", message, err)
 	}
 }
+
+type JSONRPCError struct {
+	Code    int
+	Message string
+	Data    string
+}
+
+func (e *JSONRPCError) Error() string {
+	return fmt.Sprintf("JSONRPC Error - Code: %d, Message: %s, Data: %s", e.Code, e.Message, e.Data)
+}
```

Comment:
Q: What is this used by? I don't see any code in this PR that links to `JSONRPCError`

Reply:
You are right, I also think it is safe to remove it as I couldn't also find any usage for it anywhere else.

Reply:
Tracking issue #3600